cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -175,6 +178,12 @@
 
     private static final String NAME_SUFFIX = "-kafka";
 
+    private static final String KAFKA_JMX_SECRET_SUFFIX = NAME_SUFFIX + "-jmx";
+    private static final String SECRET_JMX_USERNAME_KEY = "jmx-username";
```

Comment:
Would a general `username` and `password` be better names for the keys? (just wondering ... feel free to say _no_ ;-))